src/sections/References.html

```diff
@@ -0,0 +1,30 @@
+<section class="flex flex-col items-end gap-0 pt-20 pb-20 pl-2 pr-16 text-2xl md:gap-3 lg:gap-3 bg-customYellow md:p-32 lg:p-20 rounded-b-md font-outfit gap pt">
```

Comment:
I think if padding top and padding bottom have the same padding you can change into a single class: 'py'.